aws_kms_key.tf

```diff
@@ -22,19 +22,6 @@ resource "aws_kms_key" "kmsKeyNoDeletionWindow" {
 
 
 resource "aws_kms_key" "kmsKeyExposedPolicy" {
-  description         = "KMS key 2"
-    policy = <<EOF
-{
```

Comment:
**Accurics remediation**
 
**[HIGH]: Update access policy of KMS key**
 
**Vulnerability**
Identify any publicly accessible AWS Key Management Service master keys and update their access policy in order to stop any unsigned requests made to these resources.
 
**Remediation**
Allowing anonymous access to your AWS KMS keys is considered bad practice and can lead to sensitive data leakage. One common scenario is when an AWS user grants permissions to everyone for using the KMS key but forgets adding the Condition clauses to the key policy in order to filter the access to certain accounts.

source: accurics-